### PR TITLE
Increase timeout and parallelism for private cluster tests.

### PR DESCRIPTION
**Pull request by mm4tt**
The test timed out with the current settings.

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 1]
```diff
@@ -566,5 +567,6 @@ periodics:
           - --cluster=private-gce-test
           # TODO(mm4tt): Start setting --ssh-proxy-instance-name= when it's supported in kubetest
           - --test_args=--ginkgo.flakeAttempts=2 --ginkgo.skip=\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[DisabledForLargeClusters\] --minStartupPods=8 --node-schedulable-timeout=90m
-          - --timeout=120m
+          - --ginkgo-parallel=5
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Can you please sort them alphabetically (we're trying to keep the sorted).

Comment (mm4tt) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Done.

**Pull request merged**